### PR TITLE
Apply use context

### PR DESCRIPTION
**Pull request by ykim9611**
- applied useContext to intake the current user id.
- added user id constraints to get and post requests.

### CODE REVIEW

**Review by xanzadu — APPROVED** [review]

File: client/components/Tracker/Form/Form.jsx [hunk 1]
```diff
@@ -1,30 +1,23 @@
...
-const Form = ({ getList }) => {
+const Form = ({ getList, renderCount, user }) => {
   const [feeling, updateFeeling] = useState('');
   const [thoughts, updateThoughts] = useState('');
-  const user = useUser();
-  console.log('useContext -> ', user);
-
-
 
   const handleSubmit = (event) => {
```
Comment (jessicaawhy) [inline client/components/Tracker/Form/Form.jsx:9]: One comment- after a submitting, should the inputs be cleared?

Comment (jessicaawhy) [inline client/components/Tracker/Form/Form.jsx:9]: One comment- after a submitting, should the inputs be cleared?

File: server/routes/tracker.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const router = express.Router();
 
 router.get('/example', trackerController.exampleGet);
 
-router.get('/getList', trackerController.getList);
+router.get('/getList/:renderAmount/:user', trackerController.getList);
```
Comment (jessicaawhy) [inline server/routes/tracker.js:8]: I think we can remove the example route & controller function now! Also consider a restful implementation in the future.

**Review by jessicaawhy — APPROVED** [review]
Great job implementing user into the tracker. Approved.

File: client/components/Tracker/Form/Form.jsx (hunk 1 above)
Comment (ykim9611) [reply client/components/Tracker/Form/Form.jsx:9]: Will implement that feature in the near future.

File: server/routes/tracker.js (hunk 2 above)
Comment (ykim9611) [reply server/routes/tracker.js:8]: WIll keep in mind and update it to be a more restful route in the near future.

**Pull request merged**